app/views/content/life-as-a-teacher/age-groups-and-specialisms/early-years-teachers.md

```diff
@@ -42,14 +42,21 @@ calls_to_action:
       - Cost: "Up to $earlyyears_fees_undergrad$ per year"
         Salary: "No"
         Length of training: "3 or 4 years"
-        Qualification: "Early years teacher status"
+        Qualification: "Early years teacher status and a bachelor’s degree"
   EY-self-funded:
     name: feature_table
     arguments:
       - Cost: "This could be around $earlyyears_fees_selffundedlow$ or $earlyyears_fees_selffundedhigh$ depending on the provider"
         Salary: "No"
         Length of training: "3 months"
         Qualification: "Early years teacher status"
+  EY-apprenticeship:
+    name: feature_table
+    arguments:
+      - Cost: "Your course fees of up to $earlyyears_fees_apprenticeship$ are covered through the employer’s apprenticeship levy payments or co-funded with the government contributing 95% of the cost and employers contributing 5% of the cost. You do not need to do anything to access this funding."
```

Comment:
```suggestion
      - Cost: "Your course fees of up to $earlyyears_fees_apprenticeship$ are covered by the Department for Education. You do not need to do anything to access this funding."
```

I think this is probably too much detail for candidates - they just need to know they are not paying?

NB I've suggested saying 'covered by DfE' because I think that would cover both the levy and the co-funding options (and it is also consistent with other routes) - but I don't know this for sure so this need checking!